setup/README.md

```diff
@@ -2,7 +2,7 @@
 
 This guide is intended to get a FFXIV player setup with ACT and an overlay for parsing purposes and be able to upload logs to the FFLogs website.
 
-*Last updated: 2/11/2020*
+*Last updated: 29 JUN 2023*
```

Comment:
```suggestion
*Last updated: 2023-06-29*
```

Can we ISO 8601 here?

Reply:
Either way is better than the regionally-ambiguous format from before.  (This really should just be updated automatically on push)